web/src/shared/components/SeqGroupInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { Table as SUITable } from 'semantic-ui-react'
+
+import _ from 'lodash'
+import { GraphQlSequencingGroup, GraphQlAssay } from '../../__generated__/graphql'
+import Table from './Table'
+import { DeepPartial } from '../utilities/deepPartial'
+import AssayInfo from './AssayInfo'
+
+const excludedSequenceGroupFields = ['id', '__typename']
```

Comment:
This is gonna be petty, but I'm trying to be precise and consistent because we had a few issues earlier. 
It's "Sequenc*ing*Group", and the word "Sequence" should never appear in metamist. Maybe I should make a pre-commit hook haha

Reply:
Fixed - I will double check if I used SequenceGroup anywhere else